polyfills/_mutation/polyfill.js

```diff
@@ -1,22 +1,14 @@
 // http://dom.spec.whatwg.org/#mutation-method-macro
-function _mutation(nodes) { // eslint-disable-line no-unused-vars
-	if (!nodes.length) {
-		throw new Error('DOM Exception 8');
-	} else if (nodes.length === 1) {
-		return typeof nodes[0] === 'string' ? document.createTextNode(nodes[0]) : nodes[0];
-	} else {
-		var
-		fragment = document.createDocumentFragment(),
-		length = nodes.length,
-		index = -1,
-		node;
-
-		while (++index < length) {
-			node = nodes[index];
+function _mutation(nodes) {
+	if (nodes.length === 1) {
```

Comment:
apparently no, it does nothing on chrome and the webplatform tests https://github.com/w3c/web-platform-tests/blob/master/dom/nodes/ChildNode-after.html#L14 allow it

for adding tests, could you point me in which folder please?